### PR TITLE
Renamed Philippines to Filipino to reflect the language

### PR DESCRIPTION
**Pull request by arkiv2**
renamed:    Philippines/bago/bago.md -> Filipino/bago/bago.md
	renamed:    Philippines/bago/hakbang/hakbang.md -> Filipino/bago/hakbang/hakbang.md
	renamed:    Philippines/bagong simula/bagong_simula.md -> Filipino/bagong simula/bagong_simula.md
	renamed:    Philippines/bagong simula/ipis/ipis.md -> Filipino/bagong simula/ipis/ipis.md
	renamed:    Philippines/duterte/duterte.md -> Filipino/duterte/duterte.md
	renamed:    Philippines/movie/movie.md -> Filipino/movie/movie.md
	renamed:    Philippines/salamat.md -> Filipino/salamat.md

### CODE REVIEW

**Comment by arkiv2** [comment]
Updated pull request by adding new commits to filipino-adventure branch

**Comment by arkiv2** [comment]
Updated pull request. Ready to merge @casey-collab

**Comment by casey-collab** [comment]
It looks like this pull request deletes quite a few lines. Was this a mistake?

File: Filipino/bagong simula/ipis/ipis.md [hunk 1]
```diff
@@ -4,6 +4,6 @@ iyong kamay pra subukang takpan ang iyong mata. Sa pag angat mo ng kamay mo,
 napansin mong may kakaibang hugis na mistulang nakadikit sa iyong palad.
 Sa puntong iyon, nalaman mo na hindi lang ikaw na nasa loob ng kwarto.
 
-Subukang [durugin ang ipis](http://www.publicdomainpictures.net/pictures/60000/velka/cockroach.jpg).
+Subukang [durugin ang ipis](durog/durog.md).
```
Comment (arkiv2) [inline Filipino/bagong simula/ipis/ipis.md]: I've deleted the last path used by this story to reflect the next story which I've made and added the deleted link to my story

File: Filipino/bagong simula/ipis/ipis.md [hunk 2]
```diff
@@ -4,6 +4,6 @@ iyong kamay pra subukang takpan ang iyong mata. Sa pag angat mo ng kamay mo,
...
 
-Tigtigin ang kamay para maalis ang ipis at [kumaripas pabalas ng kwarto] (../../salamat.md).
\ No newline at end of file
+Tigtigin ang kamay para maalis ang ipis at [kumaripas palabas ng kwarto] (../../salamat.md).
```
Comment (arkiv2) [inline Filipino/bagong simula/ipis/ipis.md]: Here, I've corrected some typo. "pabalas" to "palabas" which means "going out"

**Comment by arkiv2** [comment]
I've just realized that casey, here, is a bot. Literally laughing at myself now.

**Comment by arkiv2** [comment]
Pushed some commits. Pull request is now ready for merging, @casey-collab

**Comment by casey-collab** [comment]
It looks like this pull request deletes quite a few lines. Was this a mistake?

**Pull request closed without merging**